frontends/web/src/routes/settings/settings.tsx

```diff
@@ -272,6 +287,28 @@ class Settings extends Component<Props, State> {
                               {t('settings.expert.electrum.title')}
                             </SettingsButton>
                           </div>
+                          { version !== undefined ? (
+                            <div>
+                              <h3 className="subTitle">{t('settings.info.title')}</h3>
+                              <div className="box slim divide m-bottom-large">
+                                <SettingsItem
+                                  optionalText={version}
+                                  primaryText={t('settings.info.version')}>
+                                  { update !== null && update !== undefined ? (
```

Comment:
@thisconnect didn't we have some sort of 'loading' component that can be used a placeholder while `update === undefined`? If it is undefined it means we don't know yet if there is an update, not that it is up to date.

Reply:
yes the component is `<Skeleton />` 

also `update !== null && update !== undefined ? (` could be changed to:

 `{ update != null ? (` or  just `{ update ? (`

Reply:
if the skeleton is to be used it would be more like `update === undefined ? <skeleton> : update === null ? <up to date> : <update>`? 

a bit unergonomic to use skeleton i guess, i wonder if there is a better way. maybe sth like `<Skeleton loading={update === undefined}> ... </Skeleton>`, but it's not great either

Reply:
```jsx
{update === undefined ? (
  <Skeleton />
) : ( 
  update === null ? (
    <up to date>
  ) : (
    <update>
  )
)}
```
or maybe some switch, but maybe that can't be in render and has to be in a helper function or in its own component

```jsx
{ switch(update) {
    case undefined:
       <Skeleton />
       break;
    case null:
       <up-to-date>
       break;
    default:
       <update>
} }
```

Reply:
Added a check for both `version` and `update` to be defined. If they are not, `<Skeleton>` is showed